src/test/java/org/takes/http/BkBasicTest.java

```diff
@@ -86,6 +102,59 @@ public void handlesSocket() throws IOException {
     }
 
     /**
+     * BkBasic can handle HTTP persistent connections.
+     * @throws Exception If some problem inside
+     * @checkstyle ExecutableStatementCountCheck (2 lines)
+     */
+    @Test
+    @SuppressWarnings("PMD.DoNotUseThreads")
+    public void handlesPersistentConnection() throws Exception {
+        final int port = 8080;
+        final String uri = String.format("http://localhost:%d", port);
+        final Take take = new Take() {
+            @Override
+            public Response act(final Request req) throws IOException {
+                return new TkEmpty().act(req);
+            }
+        };
+        final Thread app = new Thread(
```

Comment:
@mstzn36 you can use `FtRemote` instead running BkBasic in the thread
check https://github.com/yegor256/takes/blob/master/README.md
or https://github.com/yegor256/takes/blob/master/src/test/java/org/takes/facets/auth/social/PsGithubTest.java
or any another test using `FtRemote`